src/x265.mk

```diff
@@ -0,0 +1,65 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := x265
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 1.8
+$(PKG)_CHECKSUM := d41a3f0cc06dfc3967f6c47f458cb30b6aaa518f86c56c147946395bfe22b6f2
+$(PKG)_SUBDIR   := x265_$($(PKG)_VERSION)
+$(PKG)_FILE     := x265_$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://ftp.videolan.org/pub/videolan/$(PKG)/$($(PKG)_FILE)
+$(PKG)_DEPS     := gcc cmake yasm
+
+define $(PKG)_UPDATE
+    $(WGET) -q -O- https://ftp.videolan.org/pub/videolan/x265/ | \
+    $(SED) -n 's,.*">x265_\([0-9][^<]*\)\.t.*,\1,p' | \
+    tail -1
+endef
+
+define $(PKG)_BUILD
+    cd '$(1)' && mkdir -p 8bit 10bit 12bit
+
+    # 12 bit
+    test '$(TARGET)' = 'x86_64-w64-mingw32.static' && X265_ASM=ON || X265_ASM=OFF; \
```

Comment:
The HIGH_BIT_DEPTH options are not supported on 32-bit, do we really want to enable them?

http://hg.videolan.org/x265/file/f8b8ebdc5457/source/CMakeLists.txt#l302